static/app/components/core/button/index.chonk.tsx

```diff
@@ -51,7 +51,9 @@ function chonkSizeMapping(size: ButtonProps['size']): ChonkButtonSize {
 }
 
 export function getChonkButtonStyles(
-  p: ButtonProps & {theme: DO_NOT_USE_ChonkTheme}
+  p: Pick<ButtonProps, 'size' | 'priority' | 'busy' | 'disabled' | 'borderless'> & {
+    theme: DO_NOT_USE_ChonkTheme;
+  }
```

Comment:
changed the interface here to be as minimal as possible - only what gets really used, so that I can re-use it in the `segmentgedControl`